diff -p -up etoys-4.1.2388/Makefile.in.orig etoys-4.1.2388/Makefile.in
--- etoys-4.1.2388/Makefile.in.orig	2010-10-10 00:45:00.000000000 -0300
+++ etoys-4.1.2388/Makefile.in	2010-10-10 00:49:51.000000000 -0300
@@ -20,7 +20,7 @@ pos=$(wildcard po-new/etoys/*.po)
 
 translation: Content/locale
 
-Content/locale: Content $(pos)
+Content/locale: $(pos)
 	for dir in po-new/* ; do \
 		lang=`basename $$dir` ; \
 		for po in $$dir/*.po ; do \
@@ -33,28 +33,6 @@ Content/locale: Content $(pos)
 	done
 	touch $@
 
-# install regular etoys
-
-content_url = http://etoys.squeak.org/svn/trunk/Etoys
-po_url = http://etoys.squeak.org/svn/trunk/po
-content_rev = $(shell cat Content.revision)
-content_host = $(shell svn info Content 2>/dev/null | grep ^URL | sed 's,.*//,,;s,/.*,,')
-
-Content: Content.revision
-	if [ "$(content_host)" = "etoys.laptop.org" ] ; then \
-		mv Content Content-old ; \
-	fi
-	if [ ! -d Content ] ; \
-	then svn checkout -r $(content_rev) $(content_url) Content ; \
-	     svn checkout -r $(content_rev) $(po_url) po-new ; \
-	else svn update -r $(content_rev) Content ; \
-	     svn update -r $(content_rev) po-new ; \
-	fi
-	touch $@ -r $<
-
-ChangeLog: Content
-	./mkChangeLog
-
 etoys = Content/etoys.image     \
 	Content/etoys.changes \
 	Content/EtoysV4.stc \
@@ -65,7 +43,7 @@ examples = $(wildcard \
 	Content/ExampleEtoys/*.gif \
 	Content/ExampleEtoys/*.ogg)
 
-docs = README INSTALL LICENSE NOTICE ChangeLog Content/NEWS
+docs = README INSTALL LICENSE NOTICE Content/NEWS
 
 etoysdir = $(datadir)/etoys
 mimedir  = $(datadir)/mime/packages
@@ -168,7 +146,7 @@ etoys-rebuild: etoys-kill
 
 etoys_ver = $(shell rpm -q --queryformat '%{VERSION}' --specfile etoys.spec)
 
-etoys-src: Content Content/locale ChangeLog configure
+etoys-src: Content Content/locale configure
 	-rm -rf etoys-$(etoys_ver) etoys-$(etoys_ver).tar.gz
 	mkdir etoys-$(etoys_ver)
 	tar cf - --no-recursion `git ls-files|grep -v ^po/` \
